sdks/java/io/amazon-web-services2/build.gradle

```diff
@@ -65,6 +65,10 @@ dependencies {
   implementation "commons-lang:commons-lang:2.6"
   implementation library.java.commons_lang3
   implementation library.java.commons_codec
+  permitUnusedDeclared library.java.everit_json_schema
+  provided library.java.everit_json_schema
+  permitUnusedDeclared library.java.avro
+  provided library.java.avro
```

Comment:
Wondering @pabloem, why declaring provided dependencies if not used?
Also, regarding `provided` ... how about validating in `expand` that the necessary dependencies are on the classpath to prevent random failures at runtime if users forget to declare these?